model/src/main/java/org/mskcc/cmo/metadb/model/MetaDbPatient.java

```diff
@@ -22,16 +22,16 @@
     @Id @GeneratedValue(strategy = UuidStrategy.class)
     @Convert(UuidStringConverter.class)
     private UUID metaDbPatientId;
-    private String investigatorPatientId;
```

Comment:
renamed because it's cmoPatientId in the sample manifest, not investigator patient id